src/app/digital/internal/views/DigitalCircuitView.ts

```diff
@@ -24,6 +25,7 @@ export class DigitalCircuitView extends CircuitView {
         if (!this.assemblers) {
             this.assemblers = {
                 "ANDGate": new ANDGateAssembler(this.circuit, this, this.selections, this.sim),
+                "ORGate": new ORGateAssembler(this.circuit, this, this.selections, this.sim),
```

Comment:
nit: put an extra space before the `new` so that it's vertically aligned with the others